src/translations/nbTranslations.json

```
@@ -2236,6 +2233,9 @@
     "user_dialog": "Brukerdialog",
     "your_tasks": "Oppgaver du behandler: {{count}}"
   },
+  "no_search_results_found_with_search": "<span><icon /><h2>Ingen treff</h2></span><p>Vi fant dessverre ikke noe som passer med søket ditt.</p>",
```

rename til "heading"?
```suggestion
  "no_search_results_found_with_search": "<span><icon /><heading>Ingen treff</heading></span><p>Vi fant dessverre ikke noe som passer med søket ditt.</p>",
```